middleware/jwt/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "gotham_middleware_jwt"
-version = "0.3.1"
+version = "0.4.0-dev"
```

Comment:
Same comment on this as the other version; not sure if we need to align them.

Reply:
I've set this back - but bumped the patch version as the gotham dependency version has changed.